FhirToDataLake/test/Microsoft.Health.Fhir.Synapse.DataWriter.UnitTests/DataWriterTests.cs

```diff
@@ -42,7 +42,7 @@ public async Task GivenAValidParameter_WhenWritingToBlob_CorrectContentShouldBeW
             var streamData = new StreamBatchData(stream, 1, TestResourceType);
             await dataWriter.WriteAsync(streamData, "mockJob", 0, 0, _testDate);
 
-            var containerClient = new AzureBlobContainerClientFactory(new DefaultTokenCredentialProvider(new NullLogger<DefaultTokenCredentialProvider>()), new NullLoggerFactory()).Create(LocalTestStorageUrl, TestContainerName);
+            var containerClient = new AzureBlobContainerClientFactory(new DefaultTokenCredentialProvider(new NullLogger<DefaultTokenCredentialProvider>()), Options.Create(new StorageConfiguration()), new NullLoggerFactory()).Create(LocalTestStorageUrl, TestContainerName);
```

Comment:
Maybe create a static **StorageConfiguration** for this test file.

Reply:
Thanks, fixed it.